grading/main_configure.cpp

```diff
@@ -319,11 +320,13 @@ int main(int argc, char *argv[]) {
     }
     //container name only matters if we try to get the commands for this testcase.
     std::string container_name = "";
-    TestCase tc(config_json,which_testcase,container_name);
-    if (tc.isSubmissionLimit()) {
-      max_submissions = tc.getMaxSubmissions();
+    nlohmann::json my_testcase_json = *itr;
+    TestCase my_testcase(my_testcase_json, test_id, container_name);
+    if (my_testcase.isSubmissionLimit()) {
+      max_submissions = my_testcase.getMaxSubmissions();
     }
-    all.push_back(printTestCase(tc));
+
+    all.push_back(printTestCase(my_testcase));
```

Comment:
this is going away with your refactor to merge build & complete configs right???
because i think we're missing itempool testcases...